sigstore/_verify.py

```diff
@@ -212,6 +213,8 @@ def verify(
             san_ext = cert.extensions.get_extension_for_class(SubjectAlternativeName)
             if expected_cert_identity not in san_ext.value.get_values_for_type(
                 RFC822Name
+            ) and expected_cert_identity not in san_ext.value.get_values_for_type(
+                UniformResourceIdentifier
             ):
```

Comment:
This is good for now, but eventually we'll probably want to break this out into a `utils` helper (since we also want to check "username" typed SANs, which are identified by the Sigstore-specific `OtherName` OID).

Ref: https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md#1361415726417--othername-san

Reply:
[b7c5a99](https://github.com/sigstore/sigstore-python/pull/288/commits/b7c5a99e64b4b466b0b71a54d443e509a032a5d9) and [5247098](https://github.com/sigstore/sigstore-python/pull/288/commits/52470980791346a459eac09f6fa27393693e9d09)